### ISSUE TITLE
[Functional Bug]: Bestehende Links von externen Websiten zum cda generieren Fehlermeldung

### ISSUE THREAD

**Issue by CranacharchivGunnar** [lucascranach/cda-orga#42]
### URL of the buggy page?

https://www.th-koeln.de/kulturwissenschaften/cranach-digital-archive_12436.php

### Contact Details

gunnar.heydenreich@th-koeln.de

### What happened?

Wenn ich z.B. auf der Site
https://www.th-koeln.de/kulturwissenschaften/cranach-digital-archive_12436.php

auf den Link www.lucascranach.org 
klicke, dann kommt eine Fehlermeldung
"Warnung. Mögliches Sicherheitsrisiko erkannt..."

Die gleiche Meldung erhalte ich auch, wenn ich auf anderen externen Websiten den link zum cda anklicke...

**Comment by cnoss** [comment]
Guck mal jetzt bitte … Problem ist das falsche Protokoll (http statt https) und der www im Link. Hab eine weiterleitung eingerichtet. Achtung cache: der ist hier mitunter sehr hartnäckig.

**Comment by dangoerr** [comment]
Allerdings landet man noch auf der alten Seite, die zuvor das Projekt vorstellte. Könntest Du daher bitte auch von 
https://lucascranach.org/index.php/luther umleiten zu https://lucascranach.org/index.php/luther/einleitung ?
Engl.: https://lucascranach.org/Home/index.php/luther zu https://lucascranach.org/Home/index.php/luther/einleitung
Vielen Dank im Voraus

**Comment by dangoerr** [comment]
Ahhhh, nein, das "Home" ist mal wieder an falscher Position. So ist richtig:
https://lucascranach.org/index.php/luther zu https://lucascranach.org/index.php/luther/einleitung ?
https://lucascranach.org/index.php/Home/luther zu https://lucascranach.org/index.php/Home/luther/einleitung
Sorry!

**Comment by cnoss** [comment]
Das geht nicht, bzw. dann würde man nie auf die Seite https://lucascranach.org/index.php/luther kommen … oder ist das so gewünscht?

**Comment by dangoerr** [comment]
Ja, ist so gewünscht. https://lucascranach.org/index.php/luther ist die alte Luther-Seite, deren Inhalte inzwischen überholt sind. Falls jemand also etwa bei Projektvorstellung vor 4 Jahren zu uns verlinkt hat, landet er auf der alten Seite. Werden nicht so viele Stellen sein, aber wenn die Umleitung keine große Umstände macht, wäre es doch sinnvoll.

**Comment by CranacharchivGunnar** [comment]
Daniel, ich hab da nicht den Überblick, bitte sag Du, ggf. besser
belassen?!
G




>>> Christian Noss ***@***.***> 25.05.22 17.46 Uhr >>>

 Das geht nicht, bzw. dann würde man nie auf die Seite
https://lucascranach.org/index.php/luther kommen … oder ist das so
gewünscht?
 —
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID:
***@***.***>
